tools/process-disclosure/contains-a-valid-directory.js

```diff
@@ -14,22 +14,28 @@ const vulnerabilityJson = await fs
     );
   });
 
+// Get required key values
 const packageId = vulnerabilityJson.PackageVulnerabilityID
 const packageName = vulnerabilityJson.Package.Name
 const packageRegistry = vulnerabilityJson.Package.Registry
 
-const validDir = new RegExp(`bounties\/${packageRegistry}\/${packageName}\/${packageId}\/(vulnerability\.json|README\.md)`, 'g');
-const illegalDir = diff.filter((item) => !item.path.test(validDir));
+// Construct valid paths
+const vulnerabilityDir = `bounties/${packageRegistry}/${packageName}/${packageId}/vulnerability.json`
+const readmeDir = `bounties/${packageRegistry}/${packageName}/${packageId}/README.md`
+
+// Filter paths that do not match vulnerability valuess
+const illegalDir = diff.filter((item) => !(item.path == vulnerabilityDir || item.path == readmeDir));
 if (illegalDir.length > 0)
-    core.setFailed("Bounty path does not match vulnerability.json value.");
+    core.setFailed("Bounty path does not match vulnerability.json values.");
```

Comment:
Would it not be better to simply compare `process.env.BOUNTY_DIR` and `bounties/${packageRegistry}/${packageName}/${packageId}`?

Reply:
Simplifies things a lot more, implemented, cheers 👻

Reply:
`process.env.BOUNTY_DIR` is not necessarily true, this process will validate this 👻